### PR TITLE
Add Altair to the default user image

### PR DESCRIPTION
**Pull request by yuvipanda**
Re-arrange Dockerfile to reduce rebuilding of our R packages

Fixes https://github.com/2i2c-org/pilot-hubs/issues/478

### CODE REVIEW

**Comment by yuvipanda** [comment]
I built and pushed the images.

File: images/user/Dockerfile [hunk 1]
```diff
@@ -120,23 +120,24 @@ RUN sed -i -e '/^R_LIBS_USER=/s/^/#/' /etc/R/Renviron && \
...
 COPY Rprofile.site /usr/lib/R/etc/Rprofile.site
 # RStudio needs its own config
 COPY rsession.conf /etc/rstudio/rsession.conf
 
-# Install IRKernel
-RUN r -e "install.packages('IRkernel', version='1.1.1')" && \
-    r -e "IRkernel::installspec(prefix='${CONDA_DIR}')"
-
 # Install R packages, cleanup temp package download location
+# Install R packages before python packages, since these take longer to install
```
Comment (GeorgianaElena) [inline images/user/Dockerfile:129]: Great idea!

File: hub-templates/basehub/values.yaml [hunk 2]
```diff
@@ -106,7 +106,7 @@ jupyterhub:
       hub.jupyter.org/node-purpose: user
     image:
       name: set_automatically_by_automation
-      tag: e3bbe25
+      tag: "135823b"
```
Comment (GeorgianaElena) [inline hub-templates/basehub/values.yaml:109]: Probably a stupid question, but curious if the `"`around the image tag make any difference here?

Comment (yuvipanda) [reply hub-templates/basehub/values.yaml:109]: https://github.com/helm/helm/issues/1694 happens whenever a hash is either just a number, or a string of the form `123e44` (it is taken as a number in scientific notation). So I started putting quotes around tags as a precaution...

Comment (GeorgianaElena) [reply hub-templates/basehub/values.yaml:109]: Interesting. Thanks for explaining this!

**Comment by GeorgianaElena** [comment]
Great! Thanks @yuvipanda!

**Pull request merged**